specs/lib/loadObjSpec.js

```diff
@@ -696,4 +709,12 @@ describe("loadObj", () => {
       thrownError.message.startsWith("ENOENT: no such file or directory")
     ).toBe(true);
   });
+
+  it("vertexPattern should recognize a definition with tabs", async () => {
+    expect(loadObj.vertexPattern.exec("v -19.087 		-17 		0.343")).not.toBeNull();
+  });
+
+  it("facePattern should recognize a definition with tabs", async () => {
+    expect(loadObj.facePattern.exec("f 55 	112 	117")).not.toBeNull();
+  });
```

Comment:
I think the first test should cover the functionality well enough. Could you remove these two tests and remove the `vertexPattern` and `facePattern` exports?